.travis.yml

```diff
@@ -0,0 +1,11 @@
+sudo: false
+language: ruby
+cache:
+  directories:
+    - vendor/bundle
+    - .gradle
+rvm:
+  - jruby-1.7.24
+jdk:
+  - oraclejdk8
+script: rake test:install-core && rake test:core-fail-fast
```

Comment:
@ph I believe that was the old behavior. I'm glad to switch it, slow failure makes more sense to me. Also, I'll have it use verbose rspec formatting.